3rdParty/jemalloc/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ if (LINUX OR DARWIN)
   if (USE_JEMALLOC_PROF)
       if (USE_LIBUNWIND)
           SET(JEMALLOC_PROF "--enable-prof" "--enable-prof-libunwind" "--with-static-libunwind=${LIBUNWIND_LIB}")
-          SET(JEMALLOC_C_FLAGS "${CMAKE_C_FLAGS} -isystem ${LIBUNWIND_HOME}/include")
+          SET(JEMALLOC_C_FLAGS "${CMAKE_C_FLAGS} -isystem ${LIBUNWIND_HOME}/include -I${LIBUNWIND_HOME}/include")
```

Comment:
I don't think it's good idea, because -isystem was used to avoid warnings produced by jemalloc headers

What clang doesn't like?